src/batch/src/task/task_manager.rs

```diff
@@ -180,4 +201,41 @@ mod tests {
             .to_string()
             .contains("can not create duplicate task with the same id"));
     }
+
+    #[tokio::test]
+    async fn test_task_aborted() {
+        let manager = BatchManager::new();
+        let plan = PlanFragment {
+            root: Some(PlanNode {
+                children: vec![],
+                identity: "".to_string(),
+                node_body: Some(NodeBody::GenerateSeries(GenerateSeriesNode {
+                    start: Some(make_i32_literal(1)),
+                    // This is a bit hacky as we want to make sure the task lasts long enough
+                    // for us to abort it.
```

Comment:
Not sure what better ways there are for writing the test as we do not want to make the `abort_task` call wait until the task aborts successfully.